.gitignore

```diff
@@ -3,5 +3,6 @@
 *~
 *.swp
 *.cur_trans
+*.vtk
```

Comment:
We have `.vtk` files in `/examples/data` ignoring this extension could cause problems